src/main/java/com/openshift/internal/restclient/DefaultClient.java

```diff
@@ -109,18 +109,29 @@ public IWatcher watch(String namespace, IOpenShiftWatchListener listener, String
 	}
 
 	@Override
+	public IWatcher watch(IOpenShiftWatchListener listener, String...kinds) {
+		return this.watch("", listener, kinds);
```

Comment:
We do this at work. We USE the kubernetes-model package and manual http code since it is not supported by this library.